CHANGELOG.md

```diff
@@ -11,11 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   convenience constructor
 - Validators for `Campaign` attributes
 _ `_optional` subpackage for managing optional dependencies
+- Acquisition function for active learning: `qNIPV`
 
 ### Changed
 - Passing an `Objective` to `Campaign` is now optional
 - `GaussianProcessSurrogate` models are no longer wrapped when cast to BoTorch
 - Restrict upper versions of main dependencies, motivated by major `numpy` release
+- Sampling methods in `qNIPV` and `SequentialGreedyRecommender` are now specified via 
```

Comment:
I just noticed that I forgot to put the actual deprecation info for `SequentialGreedyRecommender`. Could you add it, e.g. like

```
### Deprecations
- `SequentialGreedyRecommender` class replaced with `BotorchRecommender`
```

and then also write `BotorchRecommender` in your entry instead?

Reply:
ive also moved breaking changes to the very top